data/notebooks/ingredients.py

```diff
@@ -583,11 +611,11 @@ def change_contributor(_):
 w_contributor.observe(change_contributor, names="value")
 
 
-def change_categories(_):
+def change_ingredient_categories(_):
```

Comment:
What's the point of a function doing nothing?

Reply:
That's a residual of a previous experiment. Removing, thanks